src/components/FeatureViews/AdminPortal/PanelManager.js

```diff
@@ -141,28 +141,31 @@ function PanelManager(props) {
                         fieldInfo["ids"].push(record[key])
                     }
                 })
-                fieldInfo["editing"].push(false);
+                if(fieldInfo["editing"][index]) {
```

Comment:
This was an attempt to only set the values to false, if they weren't already true. Not sure if this is necessary

Reply:
you aren't setting the value to false, you're adding a new "false" boolean to the `fieldInfo["editing"]` array. 

You can also simplify this to:
```
if(!fieldInfo["editing"][index]){
    fieldInfo["editing"][index] = false;
}
```

Reply:
They need to be false in order for the tertiary function to return the viewRecordRow don't they? Won't it complain that the value is undefined or have an index error when there's no value for fieldInfo["editing"][index] for the other rows?

Reply:
I'm not sure what you're trying to say here (in general, avoid using "they"/"it" and specify what you're specifically referring to). 

This is how I'm understanding your code from lines 144 - 148:
1. Check if fieldInfo["editing"] is true at `index`
2. If fieldInfo["editing"] is true, then do nothing
3. If fieldInfo["editing"] is false, then add a new `false` value to the end of the fieldInfo["editing"] array

In your first comment you referenced step 1 saying
> This was an attempt to only set the values to false, if they weren't already true.

The keyword I read here was "set" - you're trying to set any non-true values to false. Based on this, I was looking for an "=" sign where you're setting the value at that index to be false. But you used `fieldInfo["editing"].push(false)` which adds a "false" value at the end of the array. This isn't the same as setting the value to false. 

Let me know if I'm missing anything here